reviews/models/seed.SQL.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-console */
+const {
+  sequelize,
+  Restaurant,
+  Review,
+  User,
+  Owner,
+  Photo,
+} = require('./schema.SQL');
```

Comment:
Not sure if you caught this yet, but you're referencing Spark's old schema and your file path is './schema.SQL.js'